tds-dll-mssql/src/main/java/tds/dll/mssql/StudentDLL.java

```diff
@@ -1758,6 +1758,23 @@ public boolean _AllowAnonymousTestee_FN (SQLConnection connection, String client
     return false;
   }
 
+  public boolean _SecureBrowserRequired_FN (SQLConnection connection, String clientname) throws ReturnStatusException {
```

Comment:
FYI, I do not think this is used in practice.  `mssql` package is for the SQL Server implementation where `ms` stands for Microsoft.  I'm not requesting a change here as the query you're doing is straight SQL but the application will not longer work on anything other than mysql.

Reply:
Agreed.  I was appeasing the compiler with this implementation.  Maybe we can agree on what to do when there are implementations to interfaces that are not in use.